### ISSUE TITLE
Typography: Margins

### ISSUE THREAD

**Issue by imphil** [fossi-foundation/fossi-foundation-web#6]
https://zeroheight.com/822235964/p/852315-typography gives some guidance on margins below headings and paragraphs, but the web guide does not discuss this topic further (from what I could find).

Applying the general guidance to the web view leads to large or massive margins (e.g., 81px below a display h1, 24px between regular paragraphs).

Can you give some guidance on what the margins are expected to be? I tried to use some reasonable values based on what I saw in the sample designs, let me know if that's OK.

**Comment by asherdub** [comment]
@imphil I have updated the spec on that page so heading spacing is 1/2 the size of the font. Minimum and maximum values are added of 12px-32px. Paragraph spacing of 24px is fine.

I have added a new rule about line length https://zeroheight.com/822235964/p/852315-typography/t/9753cc  as I noticed the paragraph on the About page is too wide for comfortable reading. Use the class .max-w-prose on the text to constrain it to 65 characters.
